src/PageContent/Calendar/Header.js

```diff
@@ -29,8 +29,8 @@ class Header extends Component {
                     alt="close"
                 />
                 <p className="header-time">
-                    {this.props.date} {this.props.startTime} -{' '}
-                    {this.props.endTime}
+                    {this.props.startTime.toString()} - +{' '}
+                    {this.props.endTime.toString()}
```

Comment:
`{moment(this.props.startTimestamp).format(StringFormat.DATE_FULL)}`
`{moment(this.props.startTimestamp).format(StringFormat.TIME)}`
`{moment(this.props.endTimestamp).format(StringFormat.TIME)}`
for each of the strings

Reply:
is there a way that has already been defined to incorporate time zones?

<img width="576" alt="screen shot 2018-04-16 at 9 07 04 pm" src="https://user-images.githubusercontent.com/16600428/38848113-2240d51e-41ba-11e8-94bb-95e7c5530ca7.png">